client/my-sites/checkout/checkout-thank-you/transfer-pending/index.tsx

```diff
@@ -0,0 +1,130 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+import * as React from 'react';
```

Comment:
I'm not an expert in bundling, but I _think_ this wildcard import might cause a lot of unnecessary parts of React to be included in the bundle. It might be better to do `import React from 'react';` which should do what you want.

Reply:
Thanks for catching that! Updated here: https://github.com/Automattic/wp-calypso/pull/55073/commits/8b6fd0e0f6abb6c9f8277140e82e5b8061ca234d